structuredefinitions/UKCore-Immunization.xml

```diff
@@ -2,11 +2,11 @@
 <StructureDefinition xmlns="http://hl7.org/fhir">
   <id value="UKCore-Immunization" />
   <url value="https://fhir.hl7.org.uk/StructureDefinition/UKCore-Immunization" />
-  <version value="2.2.0" />
+  <version value="2.3.0" />
   <name value="UKCoreImmunization" />
   <title value="UK Core Immunization" />
   <status value="active" />
-  <date value="2022-08-26" />
+  <date value="2022-12-16" />

```

Comment:
Why has this only got a version and date change with no other changes?

Reply:
Because the IG is wrong, and I made a mistake - IG lists medicaitoncode as used in immunixation, and doesn't list medication - I  opened all 5 medications and immunization, expecting 5 results when I searched for MedicationCode, and then wrongly, made version changes to them all, and then failed to save statement!

Immunization changes will be reverted, IG will be updated